section4/exercises/classMethods.js

```diff
@@ -28,13 +28,30 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, toppings) {
     this.protein = protein;
     this.base = base;
+    this.toppings = toppings;
+  }
+  changeProtein(protein) {
+    this.protein = protein;
+  }
+  addTopping(topping) {
+    this.toppings.push(topping);
```

Comment:
Well done finding the array prototype method `push`. Experimenting with methods like this in repl is a great way to build your knowledge of the kinds of operations that happen in code and keep prep going for Turing.